README.rst

```diff
@@ -236,6 +240,23 @@ This installs pymodbus in your virtual environment with pointers directly to the
 Either method will install all the required dependencies
 (at their appropriate versions) for your current python distribution.
 
+------------------------------------------------------------
+Docker Compose
+------------------------------------------------------------
+
+If you would like to use this image as part of a `docker compose` project, you can provide a custom command. For example,
+you can spin the server by creating a docker compose file containing the following:
+
+```yml
+pymodbus-server:
+    container_name: pymodbus-server
+    image: riptideio/pymodbus:X.Y.Z
+    command: ["./examples/server_sync.py"]
+```
```

Comment:
@janiversen what do you think about this now? is it better explained how this can customised?